docs/sources/dashboards/previews.md

```diff
@@ -0,0 +1,112 @@
++++
+title = "Dashboard previews"
+keywords = ["grafana", "dashboard", "documentation", "previews"]
+aliases = ["/docs/grafana/latest/reference/previews/"]
+weight = 9
++++
+
+# Previews
+
+{{< figure  max-width="950px" src="/static/img/docs/dashboards/previews.jpg" animated-gif="/static/img/docs/dashboards/previews.gif" >}}
+
+> **Note:** Dashboard previews are available in Grafana 9.0+ as an opt-in beta feature. Please refer to the [permissions]({{< relref "#preview-visibility">}}) section to learn about issues with data source permissions before enabling the feature.
+
+Dashboard previews bring life to all dashboard list pages. Besides looking pretty, they can serve as an overview of all available dashboards, or as a tool to quickly find the right dashboard in case the dashboard names aren't enough on their own.
+
+Grafana users are not forced to use previews. Each user can choose to disable them and go back to the usual look and feel of list views. We save the choice in the browser's local storage to make it persistent across sessions.
+
+- [Enable dashboard previews](#enable-dashboard-previews)
+- [About the dashboard previews crawler](#about-the-dashboard-previews-crawler)
+- [Permissions](#permissions)
+
+## Enable dashboard previews
+
+1. Install the Image Renderer plugin or set up a remote rendering service. `3.4.0` is the minimum version of Image Renderer required for the dashboard preview feature. Refer to [Image rendering]({{< relref "../image-rendering/" >}}) for more information.
+2. Open the [config file]({{< relref "../administration/configuration.md#configuration-file-location" >}}) and enable the `dashboardPreviews` [feature toggle]({{< relref "../administration/configuration.md#feature_toggles" >}}).
+
+```
+[feature_toggles]
+# enable features, separated by spaces
+enable = dashboardPreviews
+```
+
+3. Save your changes. Grafana should reload automatically; we recommend restarting the Grafana server in case of any issues.
+
+Visit the dashboard search page to verify that the setup was successful. You should be able to see dashboard preview placeholders for all your existing dashboards after clicking on the _Show previews_ toggle at the top of page.
+
+{{< figure src="/static/img/docs/dashboards/previews-successful-setup.jpg"
+max-width="950px" animated-gif="/static/img/docs/dashboards/previews-successful-setup.gif" >}}
+
+Please check [Grafana server logs]({{< relref "../administration/configuration.md#log" >}}) in case you see any warnings after clicking on the toggle - errors logs produced by `dashboard_service` should provide more context.
```

Comment:
```suggestion
In case you see any warnings after clicking on the toggle, check [Grafana server logs]({{< relref "../administration/configuration.md#log" >}})for more context.
```

Reply:
applied, added a bit with logger name